src/myr/Myr.js

```diff
@@ -590,7 +637,8 @@ class Myr {
       position: this.position,
       scale: this.scale,
       rotation: this.rotation,
-      material: `color: ${this.color};  side: double;`,
+      material: `color: ${this.color};`,
```

Comment:
Why did we take out `side:double`?

Reply:
i had figured out that i put that there early on when trying to figure out a-frame's entity-component system and looking back it serves no useful purpose. it made the tests fail so i took it out.

Reply:
What side:double does it makes the element have color on both the inside and outside. To see this you can go inside a shape and see if it has color or not.